.github/workflows/CI.yml

```diff
@@ -28,7 +26,27 @@ jobs:
     name: Test
     strategy:
       matrix:
-        node: [17, 18]
+        node: [17, 18, 20]
+        os: [ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Install Node.js v${{ matrix.node }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node }}
+    - name: Install Packages
+      run: npm install --legacy-peer-deps
+    - name: Test
+      run: npm test
+
+  test-for-ts-eslint-v5:
+    name: Test
+    strategy:
+      matrix:
+        node: [18]
         os: [ubuntu-latest]
```

Comment:
A matrix is not really needed here if it's only one combination.

Reply:
Thank you for checking this PR. I fixed that.